pkg/services/network/constants.go

```diff
@@ -32,7 +32,7 @@ const (
 	SystemNamespace = "kube-system"
 
 	// legacyDefaultNetworkLabel was the label used for default networks.
-	// This is deprecated and is introduced only for smoother transitions.
-	// This will be released in a future release.
+	//
+	// Deprecated: legacyDefaultNetworkLabel will be removed in a future release.
 	legacyDefaultNetworkLabel = "capw.vmware.com/is-default-network"
```

Comment:
We probably should have a follow-up issue to figure when / how we can get rid of the capw labels - original issue is here: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/1483

Reply:
Follow-up https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2394